.github/workflows/regression-tests.yml

```diff
@@ -90,7 +90,8 @@ jobs:
       # Run tests that don't need a simulator.
     - name: Install Python testing dependencies
       run: |
-        pip install nox
+        conda install --yes sqlite=3.39
+        pip install nox==2022.8.7 virtualenv==20.16.7
```

Comment:
Are these pins necessary if sqlite is being installed? Pinning to nox 2022.8.7 means that 3.6 test will fail. Switch instead to `nox<2022.9` or similar.

Reply:
As I replied to Tomasz, I suspect it's not related to conda, but a github image issue. I suppose we could wait a bit to see if there's a bug reported upstream. Alternatively, could you re-run some previous ci runs that passed?